components/Input/NumericInput.tsx

```diff
@@ -32,7 +32,7 @@ const NumericInput: FC<NumericInputProps> = ({
 }) => {
 	const handleOnChange = (e: ChangeEvent<HTMLInputElement>) => {
 		const { value } = e.target;
-		if (!isValidNumber(value)) return;
+		if (!isValidNumber(value)) return onChange(e, '0.00');
```

Comment:
oh good point, I checked and its used in these places:

- loan amount inputs
- `StakeTab` and `DepositTab` (/earn)
- gas price selector (custom gas price)

I think the change is an improvement for the first two, since currently if you try to backspace when there is only 1 digit in the input, the input does not change/clear as expected

However for the gas price selector, this change makes choosing a custom gas price feel strange since the input field will populate with "0.00" if you delete the contents, but gas is usually a whole number...

I'll leave out this change so the PR doesn't have too much going on